cmd/server/app/server.go

```diff
@@ -142,6 +143,16 @@ func (p *Proxy) Run(o *options.ProxyRunOptions, stopCh <-chan struct{}) error {
 		defer frontendStop()
 	}
 
+	if o.IsLeaseCountingEnabled() {
+		leaseLabels, err := labels.ConvertSelectorToLabelsMap(o.LeaseLabelSelector)
+		if err != nil {
+			return fmt.Errorf("could not parse lease label selector (%q): %w", o.LeaseLabelSelector, err)
+		}
+		leaseController := leases.NewController(k8sClient, o.ServerID, int32(o.LeaseDuration.Seconds()), o.LeaseRenewalInterval, o.LeaseGCInterval, fmt.Sprintf("konnectivity-proxy-server-%v", o.ServerID), o.LeaseNamespace, leaseLabels)
+		klog.V(1).Infoln("Starting lease acquisition and garbage collection controller.")
+		leaseController.Run(ctx)
```

Comment:
+1. We also need to use a timeout on the lease to ensure that a server which went down to quickly to cleanup doesn't stick around in the count forever. However it would be good to notice faster than that.

Reply:
Added this! Let me know if what I wrote works.